package.json

```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "dev": "node esbuild.config.mjs",
     "build": "node esbuild.config.mjs production",
+    "build:dev": "node esbuild.config.mjs development",
     "lint": "eslint ./src --fix && eslint ./tests --fix && tsc --noEmit --pretty && svelte-check",
     "test": "jest --ci",
-    "test:dev": "jest --watch"
+    "test:dev": "jest --watch",
+    "test:obsidian": "pwsh -ExecutionPolicy Unrestricted -NoProfile -File ./scripts/Test-TasksInLocalObsidian.ps1"
```

Comment:
I think that the name `test:obsidian` does not really convey what it does, and in fact it confused me, as I though it was actually running tests inside obsidian.

How about `test:deploy`?

Or since it isn't running any tests, `deploy` or `dev:deploy`?

Reply:
Settled on `deploy:local`. Thanks,